### PR TITLE
Initial cleanup and Vuex installation

### PR DESCRIPTION
**Pull request by austinstarin**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 <style>
-#app {
-  font-family: 'Avenir', Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
-}
+  @import '../node_modules/bulma/css/bulma.css';
```
Comment (bgantick) [inline src/App.vue:23]: missing bulma from deps in package.json

File: src/App.vue [hunk 2]
```diff
@@ -1,23 +1,24 @@
 <template>
   <div id="app">
-    <img src="./assets/logo.png">
+    <TheHeader></TheHeader>
```
Comment (bgantick) [inline src/App.vue:3]: multi word components should be registered with dashes `<my-component>`

Comment (austinstarin) [reply src/App.vue:3]: The style guide gives the option of PascalCase or kebab-case, do you have a preference?

https://vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended

File: src/App.vue (hunk 1 above)
Comment (austinstarin) [reply src/App.vue:23]: Thank you, will resolve.

File: src/App.vue (hunk 2 above)
Comment (bgantick) [reply src/App.vue:3]: Ok, pascal case is fine. Most examples I was seeing were kebab. Should rename components to `BaseHeader` and `BaseFooter` though: https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended

**Pull request merged**